code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -156,6 +156,12 @@
 			return
 	return
 
+/obj/item/weapon/storage/pill_bottle/AltClick()
+	if(!usr.isUnconscious() && Adjacent(usr))
```

Comment:
@intigracy in fact verbs are only different to procs in that they automatically get added to the `verbs` list.